### PR TITLE
Fixes spelling and misleading messages

### PR DESCRIPTION
**Pull request by harmonyn**
I sure do love PRs that involve no actual work

### CODE REVIEW

File: code/game/machinery/newscaster.dm [hunk 1]
```diff
@@ -980,15 +980,15 @@ var/list/obj/machinery/newscaster/allCasters = list()
...
-					dat+="<I>Apart from some uninteresting Classified ads, there's nothing on this page...</I>"
+					dat+="<I>Apart from some uninteresting classified ads, there's nothing on this page...</I>"
 				if(scribble_page==curr_page)
 					dat+="<BR><I>There is a small scribble near the end of this page... It reads: \"[scribble]\"</I>"
 				dat+= "<HR><DIV STYLE='float:left;'><A href='?src=\ref[src];prev_page=1'>Previous Page</A></DIV>"
 		dat+="<BR><HR><div align='center'>[curr_page+1]</div>"
 		human_user << browse(dat, "window=newspaper_main;size=300x400")
 		onclose(human_user, "newspaper_main")
 	else
-		user << "The paper is full of intelligible symbols!"
```
Comment (XDTM) [inline code/game/machinery/newscaster.dm]: honk

File: code/game/gamemodes/nuclear/nuclear.dm [hunk 2]
```diff
@@ -255,7 +255,7 @@
 	else if (!disk_rescued &&  crew_evacuated)
 		feedback_set_details("round_end_result","halfwin - detonation averted")
 		world << "<FONT size = 3><B>Syndicate Minor Victory!</B></FONT>"
-		world << "<B>[syndicate_name()] operatives recovered the abandoned authentication disk but detonation of [station_name()] was averted.</B> Next time, don't lose the disk!"
+		world << "<B>[syndicate_name()] operatives survived the assault but failed to secure the disk or detonate [station_name()].</B> Next time, don't lose the disk!"
```
Comment (KorPhaeron) [inline code/game/gamemodes/nuclear/nuclear.dm]: Why are you changing this?

Comment (bgobandit) [inline code/game/gamemodes/nuclear/nuclear.dm]: check the requirements, the ops don't necessarily have the disk, it's just that it's not at CentCom

Comment (RandomMarine) [inline code/game/gamemodes/nuclear/nuclear.dm]: The implication is that operatives eventually dig up the disk anyway.

Comment (bgobandit) [inline code/game/gamemodes/nuclear/nuclear.dm]: maybe but it only ever results in "WTF WE DON'T HAVE THE DISK" in ooc

Comment (KorPhaeron) [inline code/game/gamemodes/nuclear/nuclear.dm]: I've only seen people react that way a couple times, and both times several people answered the way RandomMarine just did

**Comment by jughu** [comment]
Yet it isnt a bugfix so can it be gifted a token :^)

File: code/game/machinery/newscaster.dm (hunk 1 above)
Comment (ExcessiveUseOfCobblestone) [inline code/game/machinery/newscaster.dm]: smart paper

**Pull request merged**